packages/transformer/src/IModelTransformer.ts

```diff
@@ -2220,25 +2220,28 @@ export class IModelTransformer extends IModelExportHandler {
    * You generally should not call this function yourself and use [[process]] with [[IModelTransformOptions.argsForProcessChanges]] provided instead.
    * It is public for unsupported use cases of custom synchronization transforms.
    * @note if [[IModelTransformOptions.argsForProcessChanges]] are not defined in this transformation, this will fail
```

Comment:
Documentation says: `if [[IModelTransformOptions.argsForProcessChanges]] are not defined in this transformation, this will fail without setting the `initializeReverseSyncVersion` option to `true``, but as I understand it will just return early (without updating synch version).